### DISCUSSION TITLE
Log of deleted files?

### DISCUSSION THREAD

**Discussion by njgentile58** [iterate-ch/cyberduck#14594] [Q&A]
Is there a log of deleted files?  
I deleted a folder but instead of deleting folder using CyberDuck but instead of the folder being deleted about 50 files deleted before I could pull the ethernet cable from the computer.  I never moved so fast....

Mac Monterey OS 12.3.1 using CyberDuck 8.5.9

I see the transfer window, but can't find any option to get that list of deleted  files.  Or a general activity log?
I am grateful for help!
Ideas?

**Comment by dkocher** [comment]
There is no log maintained. You will have to check if your storage supports versioning that would allow to retrieve deleted files. Refer to our [documentation](https://docs.cyberduck.io/cyberduck/info/#versions).

**Comment by dkocher** [comment]
We are also working on a feature [^1] that would allow to retrieve deleted files on regular non-versioned storage.

[^1]: #14376
